packages/web-components/public/appointment-planner.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="utf8" />
+  <meta name="viewport" content="width=device-width" />
+
+  <title>Web Components - Appointment Planner</title>
+  <style>
+    #appointment-planner {
+      max-width: 700px;
+      margin: 0 auto;
+      margin-top: 50px;
+    }
+  </style>
+  <link rel="stylesheet" href="./dist/appointment-planner.css" />
+</head>
+
+<body>
+  <div>
+    <div id="appointment-planner"></div>
+  </div>
+  <script src="./dist/appointment-planner.js"></script>
+  <script>
+    ReapitAppointmentPlannerComponent &&
+      new ReapitAppointmentPlannerComponent({
+        theme: {
+          hoverBackgroundColor: 'white',
+        },
+        apiKey: '',
+        target: document.getElementById('appointment-planner'),
```

Comment:
Not called target now - have called it parentElement and is an id string eg `#book-a-viewing` - target got confusing because it is a reserved prop in Svelte. See #785